eip155/caip10.md

```diff
@@ -21,13 +21,33 @@ While Ethereum "accounts" were the unstated norm in the definition of
 [CAIP-10][], there are still some particularities of the syntax that should be
 specified for the unfamiliar.  
 
+### Special case of EOA
+
+To express Ethereum account without reference to any specific [EIP-155][] chain ID, CAIP-10 identifiers can use the special `chainId` segment `0`.
```

Comment:
needs link

Reply:
oh sorry this is a weird elite markdown trick that we included in the style guide a while back-- that unfamiliar linking style actually gets its referent from the alias defined [here](https://github.com/ChainAgnostic/namespaces/blame/f521f2f1bbb71869af791becdd8017b3c70ab7d7/eip155/caip10.md#L103).  Most PR authors don't notice the instructions [here](https://github.com/ChainAgnostic/namespaces/blob/main/CONTRIBUTING.md#style-guide) in the `CONTRIBUTING.md` file so I just add a commit to everyone's PR translating the normal-human links to our kooky nerd link format.